src/seed_tools/utils/diff_strings.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { exec } from 'child_process';
+import fs from 'fs/promises';
+import os from 'os';
+import path from 'path';
+
+export default async function diffStrings(
+  string1: string,
+  string2: string,
+  displayFileName1: string,
+  displayFileName2: string,
+): Promise<string> {
+  const tmpDir = await fs.mkdtemp(path.join(os.tmpdir(), 'diffString-'));
+
+  const tmpFile1 = path
+    .join(tmpDir, `file1-${Date.now()}.txt`)
+    .replaceAll('\\', '/');
+  const tmpFile2 = path
+    .join(tmpDir, `file2-${Date.now()}.txt`)
+    .replaceAll('\\', '/');
+
+  // Write strings to temporary files.
+  await fs.writeFile(tmpFile1, string1);
+  await fs.writeFile(tmpFile2, string2);
+
+  return await new Promise<string>((resolve, reject) => {
+    // Use git diff to generate readable diff.
+    exec(
```

Comment:
Why not `execa.execaSync`?
Or `util.promisify` over `exec()`?

Reply:
sadly, `execa` doesn't work in tests (esm/non-esm stuff).

replaced with `promisify(exec)`.

Reply:
removed `execa` entirely, as there's no point in keeping it. I was hoping to use it everywhere instead of `child_process`, but the fact that it cannot be used in tests (import just doesn't work), means we should not spend time with it and just use the good ol' `child_process`..

Reply:
thanks.
It also good in terms of security/